aries-site/src/pages/learn/grid-fundamentals-part-one.mdx

```diff
@@ -0,0 +1,290 @@
+import { Anchor, Notification, Paragraph } from 'grommet';
+import {
+  StepOne,
+  StepTwo,
+  StepThree,
+  StepFour,
+  StepFive,
+} from '../../examples/learn/tutorials/grid-fundamentals-part-1';
+
+In this tutorial you will use Grid to display a collection of products, similar to
+that of an e-commerce website, and use its [properties](https://v2.grommet.io/grid?theme=hpe#props)
+to:
+
+- Present a collection of items in a grid.
+- Allow the number of grid columns to adapt to a variety of screen widths.
+- Allow the items to adapt and flex within the available space.
+
+At the end of this tutorial, you will have built this:
+
+<!-- show example of product cards in a grid -->
```

Comment:
Just flagging that we need to add the final state here still

Reply:
done